srs_data/tagging.py

```diff
@@ -14,12 +15,15 @@ class ProcessTagsThatClientReturns(BaseTool):
     EXPECTED_INPUT_FILE_COLUMNS = ['text', 'tag', 'notes']
 
     def __init__(self, config):
-        self._config = config['srs_data']['tagging']
+        self._config = config['tagging']
         self._client = config['client']
         self._start_date = config['start_date']
         self._start_date = '20170903'
         self._end_date = config['end_date']
         self._input_directory = os.path.join(ASAPP_ROOT, 'data', self._client, self._start_date)
+        # this is bad. this will become more natural when we start making
```

Comment:
agreed. there should be some standard format from a constants.py file that gets pulled in and the start_date filled eventually

Reply:
Correct. However, there does need to be some programmatic logic there (which is easy and fine), e.g.

```python
def uniform_sample_file_updated_with_client_tags(self):
        return self.uniform_sample_file.replace('.csv', '_auto.csv')
```